openpathsampling/features/xyz.py

```diff
@@ -0,0 +1,22 @@
+attributes = ['xyz']
+
+def xyz(snapshot):
+    """
+    Returns
+    -------
+    xyz : numpy.ndarray, shape=(atoms, 3), dtype=numpy.float32
+        atomic coordinates without dimensions. Be careful.
+
+    Notes
+    -----
+    SERIOUS PROBLEM: whether .coordinates returns a u.Quantity or jut
```

Comment:
Is this "SERIOUS PROBLEM" in the notes still a serious problem? I think this is solved (perhaps by PRs even before this one).